podcasts/Onboarding/Plus/UpgradeLandingView.swift

```diff
@@ -0,0 +1,324 @@
+import SwiftUI
+
+struct UpgradeLandingView: View {
+    @EnvironmentObject var viewModel: PlusLandingViewModel
+
+    private let tiers: [UpgradeTier] = [.plus, .patron]
+
+    @State private var currentPage: Int = 0
+
+    @State private var displayPrice: DisplayPrice = .yearly
+
+    var body: some View {
+        VStack(spacing: 0) {
+            topBar
+
+            ZStack {
+
+                if currentPage == 0 {
+                    tiers[0].background
+                        .transition(.opacity.animation(currentPage == 0 ? .easeIn : .easeOut))
+                        .ignoresSafeArea()
```

Comment:
We'll also want to add a clipped here. I found with the `PlusBackgroundGradientView` that it would appear before the actually view when being pushed or popped. 

```suggestion
                        .ignoresSafeArea().clipped()
```

Reply:
I did this change but turned out it was clipping at the bottom, so I removed it.